cordova/config.xml

```diff
@@ -29,44 +29,51 @@
+    <icon height="32" src="res/ios/Icon-App-16x16@2x.png" width="32" />
+    <icon height="48" src="res/ios/Icon-App-16x16@3x.png" width="48" />
+    <icon height="20" src="res/ios/Icon-App-20x20@1x.png" width="20" />
+    <icon height="40" src="res/ios/Icon-App-20x20@2x.png" width="40" />
+    <icon height="60" src="res/ios/Icon-App-20x20@3x.png" width="60" />
+    <icon height="29" src="res/ios/Icon-29.png" width="29" />
+    <icon height="48" src="res/ios/Icon-48.png" width="48" />
+    <icon height="50" src="res/ios/Icon-50.png" width="50" />
+    <icon height="55" src="res/ios/Icon-55.png" width="55" />
+    <icon height="57" src="res/ios/Icon-57.png" width="57" />
+    <icon height="58" src="res/ios/Icon-58.png" width="58" />
+    <icon height="64" src="res/ios/Icon-64.png" width="64" />
+    <icon height="72" src="res/ios/Icon-72.png" width="72" />
+    <icon height="76" src="res/ios/Icon-76.png" width="76" />
+    <icon height="80" src="res/ios/Icon-80.png" width="80" />
+    <icon height="87" src="res/ios/Icon-87.png" width="87" />
+    <icon height="88" src="res/ios/Icon-88.png" width="88" />
+    <icon height="100" src="res/ios/Icon-100.png" width="100" />
+    <icon height="114" src="res/ios/Icon-114.png" width="114" />
+    <icon height="120" src="res/ios/Icon-120.png" width="120" />
+    <icon height="128" src="res/ios/Icon-128.png" width="128" />
+    <icon height="144" src="res/ios/Icon-144.png" width="144" />
+    <icon height="152" src="res/ios/Icon-152.png" width="152" />
+    <icon height="167" src="res/ios/Icon-167.png" width="167" />
+    <icon height="172" src="res/ios/Icon-172.png" width="172" />
+    <icon height="180" src="res/ios/Icon-180.png" width="180" />
+    <icon height="196" src="res/ios/Icon-196.png" width="196" />
+    <icon height="256" src="res/ios/Icon-256.png" width="256" />
+    <icon height="512" src="res/ios/Icon-512.png" width="512" />
+    <icon height="1024" src="res/ios/Icon-1024.png" width="1024" />
+    <splash src="res/screen/ios/Default@2x~universal~anyany.png"/>
     <feature name="StatusBar">
       <param name="ios-package" onload="true" value="CDVStatusBar"/>
     </feature>
     <preference name="Orientation" value="portrait"/>
     <preference name="Fullscreen" value="true"/>
-    <edit-config target="NSCameraUsageDescription" file="*-Info.plist" mode="merge" overwrite="true">
-      <string>So you can photograph items for donation.</string>
+    <edit-config target="NSCameraUsageDescription" file="*-Info.plist" mode="merge">
+        <string>So you can photograph items for donation.</string>
     </edit-config>
-    <edit-config file="*-Info.plist" mode="merge" target="NSPhotoLibraryUsageDescription">
-      <string>Needs photo library access to get photos from there</string>
+    <edit-config target="NSPhotoLibraryUsageDescription" file="*-Info.plist" mode="merge">
+        <string>need photo library access to get pictures from there</string>
```

Comment:
Do we intentionally changed text here from `Needs photo library access to get photos from there` to `need photo library access to get pictures from there` ?

Reply:
No, typo because of copy paste. Will change it back. 
Thanks for pointing it out

Reply:
Sure, Thanks :)